src/modules/colorPicker/ColorPickerUI/Helpers/Logger.cs

```diff
@@ -7,14 +7,14 @@
 using System.Globalization;
 using System.IO;
 using System.IO.Abstractions;
+using interop;
 
 namespace ColorPicker.Helpers
 {
     public static class Logger
     {
         private static readonly IFileSystem _fileSystem = new FileSystem();
-        private const string ModuleLocation = "Microsoft\\PowerToys\\ColorPicker";
-        private static readonly string ApplicationLogPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), ModuleLocation, "Logs");
+        private static readonly string ApplicationLogPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), Constants.AppDataPath(), "ColorPicker\\Logs");
```

Comment:
What about returning the full path in `Constants.AppDataPath()` so it already includes `Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData)`?

Reply:
Good catch!